src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("14.0.0.0")]
-[assembly: AssemblyFileVersion("14.1.0.0")]
+[assembly: AssemblyVersion("15.1.0.0")]
```

Comment:
Where is the code for 15.0?

Reply:
Please find 15.0 code in intermediate commit: https://github.com/Azure/azure-sdk-for-net/pull/3771/commits/c611db67198814fd32c0a645e827f02de1832d7d#diff-ccc216d74235230fc889386be30ad4abR7

Reply:
@EvgenyAgafonchikov make this 15.0.0